### PR TITLE
[ETFE-1147] Build new page for SR41: "details-select-item"

### PR DESCRIPTION
**Pull request by grahampaulcook**

### CODE REVIEW

File: app/views/DetailsSelectItemView.scala.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+        @govukButton(
+            ButtonViewModel(messages("site.continue"))
+        )
+    }
+}
+
+@heading = {
+    @h1("detailsSelectItem.heading", None, "govuk-fieldset__heading")
+}
```
Comment (james-work-account) [inline app/views/DetailsSelectItemView.scala.html:80]: To increase coverage on this view (essentially "ignore the `apply` methods for the view when running coverage")

```suggestion
}

@{
 //$COVERAGE-OFF$
}
```

File: test/controllers/SelectItemsControllerSpec.scala
Comment (james-work-account) [inline test/controllers/SelectItemsControllerSpec.scala:1]: Bunch of unused imports in this file now

File: app/views/DetailsSelectItemView.scala.html (hunk 1 above)
Comment (james-work-account) [reply app/views/DetailsSelectItemView.scala.html:80]: NotPermittedView and RefusedAmountView are missing this too - please could you add them there whilst I remember?

**Review by james-work-account — CHANGES_REQUESTED** [review]

File: test/controllers/SelectItemsControllerSpec.scala
Comment (grahampaulcook) [reply test/controllers/SelectItemsControllerSpec.scala:1]: Removed unused imports

File: app/views/DetailsSelectItemView.scala.html (hunk 1 above)
Comment (grahampaulcook) [reply app/views/DetailsSelectItemView.scala.html:80]: Added `$COVERAGE-OFF`

**Review by james-work-account — APPROVED** [review]

**Pull request merged**